### PR TITLE
Add disable Google reCAPTCHA for Admin panel Reset Password page from command-line

### PR DESCRIPTION
**Pull request by ravi-chandra3197**
## Purpose of this pull request

This pull request (PR) ...
disable Google reCAPTCHA for Admin panel Reset Password page from command-line
## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.4/security/google-recaptcha.html

## Links to Magento source code

-  ...

### CODE REVIEW

File: src/guides/v2.4/security/google-recaptcha.md [hunk 1]
```diff
@@ -30,3 +30,9 @@ The extension supports a command line option for disabling reCAPTCHA. Use this c
 ```bash
 bin/magento security:recaptcha:disable-for-user-login
 ```
+
+Can disable Google reCAPTCHA for Admin panel Reset Password page from command-line:
```
Comment (BarnyShergold) [inline src/guides/v2.4/security/google-recaptcha.md:34]:
```suggestion
You can also disable Google reCAPTCHA for Admin panel Reset Password page from the command line:
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]

**Comment by ravi-chandra3197** [comment]
Hello @BarnyShergold 
Thank you for a suggestion I have update PR as per your suggestion could you please review it?

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import pr to magento-devdocs/devdocs

**Comment by m2-github-services** [comment]
@dobooth the Pull Request is successfully imported.

**Pull request merged**